.github/workflows/daily-tests.yml

```diff
@@ -18,3 +18,5 @@ jobs:
     uses: ./.github/workflows/pull-request.yml
   run-zephyr-builds:
     uses: ./.github/workflows/zephyr.yml
+  sparse-zephyr:
```

Comment:
I'd argue some better rationale could be added to the git commit that "why not".

Reply:
Fair enough, I should probably have said something better like "there is no reason to exclude sparse from daily tests" :-)

Thanks for not blocking this.